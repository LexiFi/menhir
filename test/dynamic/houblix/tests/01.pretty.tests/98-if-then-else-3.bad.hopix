let sequence =
  if (x =? 0) { foo 0
  } else { if (x <? 0) { bar 0
    else { if (x >? 0) { baz 0
  } else { live 0 } } }