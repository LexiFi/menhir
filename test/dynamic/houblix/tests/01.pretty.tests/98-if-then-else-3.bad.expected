Line 4, characters 4-8: Error (parsing)
  Syntax error.
